kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/usb/UsbModemDevice.java

```diff
@@ -197,4 +248,87 @@ private int getDigitPosition(String devName) {
             return pos;
         }
     }
+
+    private class TtyDev {
```

Comment:
You want this nested class to be static (you don't want to mess with the outer type)

Reply:
@cdealti I agree.